query-algebrizer/src/cc.rs

```diff
@@ -13,6 +13,7 @@
 extern crate mentat_core;
 extern crate mentat_query;
 
+use std::fmt;
 use ::std::collections::BTreeMap;
```

Comment:
Could you make these styles match? I used `::std` out of some kind of misguided exploration of Rust style, I think; if your `std` works, let's do that.